jasmine-gjs.spec

```diff
@@ -46,6 +46,8 @@ be displayed in your terminal.
 
 
 %changelog
+* Mon Aug 26 2020 Andy Holmes <andrew.g.r.holmes@gmail.com> - 2.3.4-1
```

Comment:
It's Wednesday, but no worries, I'll fix it up.

Reply:
oops :D